front_end/panels/ai_chat/LLM/GoogleAIProvider.ts

```
@@ -0,0 +1,548 @@
+      return null;
+    }).filter(Boolean);
+
+    return functionDeclarations.length > 0 ? [{
+      function_declarations: functionDeclarations
+    }] : undefined;
+  }
+
+  /**
+   * Makes a request to the Google AI API
+   */
+  private async makeAPIRequest(endpoint: string, payloadBody: any): Promise<any> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)